DEVELOP.md

```diff
@@ -0,0 +1,162 @@
+# Development
+
+This document will guide you through how to
+
+. Set up the local development environment
+. Build web UI and configure backend service
+. Run the cloud gaming demo for the local development
+. Add additional games
```

Comment:
```suggestion
# Develop your own demo

This document shows how to customise the demo and add your own changes.
It describes how to build the web UI, configure the backend service, run the demo with your local changes and add additional games.
```
If we want to have a list here, it must correspond fully to the sections below. So imo it's easier to have a sentence describing the content.

Reply:
I'm fine with that.